lib/releaselib.sh

```diff
@@ -486,6 +479,9 @@ release::set_release_version () {
   if [[ ${RELEASE_VERSION[alpha]} ]]; then
     ORDERED_RELEASE_KEYS+=("alpha")
   fi
+  for key in ${!ORDERED_RELEASE_KEYS[*]}; do
```

Comment:
```suggestion
  for key in ${!ORDERED_RELEASE_KEYS[@]}; do
```

Slight preference towards `@` here to be consistent with the fixes in https://github.com/kubernetes/release/pull/1015 and the note about usage above this block.

Reply:
Good catch.  That's a copy paste error from above.